src/web/TextInput.tsx

```diff
@@ -143,7 +159,13 @@ export class TextInput extends React.Component<Types.TextInputProps, TextInputSt
             keyboardType = 'tel';
         }
 
-        return keyboardType;
+        if (returnKeyTypeProp === 'search') {
+            keyboardType = 'search';
+            wrapInForm = true;
+        }
+
+        let keyboardTypeObject = { keyboardType, wrapInForm };
```

Comment:
There's no need (or advantage) to assign this to a temporary variable. Just return the object literal directly.